### PR TITLE
Google BigQuery Destination/v2

### PR DESCRIPTION
**Pull request by erinkcochran87**
This PR updates the docs for the new Google BigQuery destination.

### CODE REVIEW

**Comment by erinkcochran87** [comment]
@ccapurso This is what I have so far for review:

- **Setup instructions**: [Preview](https://deploy-preview-375--stitchdocs.netlify.com/docs/destinations/google-bigquery/connecting-google-bigquery-to-stitch) | [File](https://github.com/stitchdata/docs/pull/375/files#diff-7cdae178d3aeba47821456e69ebc9509)

- **Pricing guide**: [Preview](https://deploy-preview-375--stitchdocs.netlify.com/docs/destinations/google-bigquery/understanding-stitch-impact-on-bigquery-costs) | [File](https://github.com/stitchdata/docs/pull/375/files#diff-6258b44bde927ccfc22bc4717c6f6415)

- **API destination form property**: [Preview](https://deploy-preview-375--stitchdocs.netlify.com/docs/stitch-connect/api#destination-form-properties-google-bigquery-object) | [File](https://github.com/stitchdata/docs/pull/375/files#diff-596837b37d895db1c03329ecba2d0933)

File: _data/destinations/reference/bigquery/v2.yml [hunk 1]
```diff
@@ -0,0 +1,383 @@
...
+
+
+# -------------------------- #
+#       Stitch Details       #
+# -------------------------- #
+
+# Details about Stitch's implementation &
+# requirements for the destination.
+
+region-list:
```
[the remark this answers is not in the document]
Comment (erinkcochran87) [reply _data/destinations/reference/bigquery/v2.yml]: Got it.

**Comment by erinkcochran87** [comment]
Added 11/5:

- **Nested data structure storage**: [Preview](https://deploy-preview-375--stitchdocs.netlify.com/docs/replication/loading/google-bigquery-storing-nested-data-structures) | [File](https://github.com/stitchdata/docs/pull/375/files#diff-f7cbdf750168bae8fb02b2248375f350)

- **Primary Keys table**: [Preview](https://deploy-preview-375--stitchdocs.netlify.com/docs/replication/loading/primary-keys-system-table) | [File](https://github.com/stitchdata/docs/pull/375/files#diff-89d2db44e5963dc2b58ae00e11eeeb8f)

**Comment by erinkcochran87** [comment]
@ccapurso I think this is the last round of reviews!

- **Reference guide**
   - **Overall**: [Preview](https://deploy-preview-375--stitchdocs.netlify.com/docs/destinations/google-bigquery/reference/v2) | [File](https://github.com/stitchdata/docs/pull/375/files#diff-88612d3ddd9b7205d97acf1bc9753d7a)
   - **GCS Regions**: [Preview](https://deploy-preview-375--stitchdocs.netlify.com/docs/destinations/google-bigquery/reference/v2#supported-gcs-regions) | [File](https://github.com/stitchdata/docs/pull/375/files#diff-1ec09b1ede85712e134cae9106d36946)
   - **Replication process**: [Preview](https://deploy-preview-375--stitchdocs.netlify.com/docs/destinations/google-bigquery/reference/v2#replication-process) | [File](https://github.com/stitchdata/docs/pull/375/files#diff-88612d3ddd9b7205d97acf1bc9753d7aR120) 
   - **Data typing**: [Preview](https://deploy-preview-375--stitchdocs.netlify.com/docs/destinations/google-bigquery/reference/v2#transformations--data-typing) | [File](https://github.com/stitchdata/docs/pull/375/files#diff-cfcfed8de5aa3a3ff37b05efe8196f20)
   - **Time data**: [Preview](https://deploy-preview-375--stitchdocs.netlify.com/docs/destinations/google-bigquery/reference/v2#transformations--timezones) | [File](https://github.com/stitchdata/docs/pull/375/files#diff-88612d3ddd9b7205d97acf1bc9753d7aR261)

File: _destinations/bigquery/v2/bigquery-overview-v2.md [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+
+          - transformation: "Convert spaces to underscores"
+            source-column: "<code>customer id</code>"
+            destination-column: &customerid2 "<code>customer_id</code>"
+
+          - transformation: "Convert special characters to underscores"
+            source-column: "<code>customer#id</code>"
+            destination-column: *customerid2
+
+          - transformation: "Remove leading non-letter characters, except for leading underscores"
```
[the remark this answers is not in the document]
Comment (erinkcochran87) [reply _destinations/bigquery/v2/bigquery-overview-v2.md]: Got it - thanks for clarifying/correcting this!

File: _destinations/bigquery/v2/bigquery-overview-v2.md [hunk 3]
```diff
@@ -0,0 +1,276 @@
+---
+# -------------------------- #
+#        Page Controls       #
+# -------------------------- #
+title: Google BigQuery v2 Destination Reference
+permalink: /destinations/google-bigquery/reference/v2
+keywords: bigquery, google bigquery, bigquery data warehouse, bigquery etl, etl to bigquery
+summary: "Google BigQuery is a fully managed, cloud-based big data analytics web service for processing very large read-only data sets. BigQuery was designed for analyzing data on the order of billions of rows, using a SQL-like syntax."
```
[the remark this answers is not in the document]
Comment (erinkcochran87) [reply _destinations/bigquery/v2/bigquery-overview-v2.md]: To be honest, I'm not sure. This was pulled from marketing copy when the original BQ loader was launched. I can update the description to be more in line with what we're currently using on the marketing site.

[the remark this answers is not in the document]
Comment (ccapurso) [reply _destinations/bigquery/v2/bigquery-overview-v2.md]: Sounds good to me.

**Pull request merged**